src/formatDate.js

```diff
@@ -50,7 +50,32 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [, , , fromSeparator] = fromFormat;
+  const [, , , toSeparator] = toFormat;
+  const dateArr = date.split(fromSeparator);
+  const dateKeys = {};
+  const finalView = [];
+
+  for (let i = 0; i < dateArr.length; i++) {
+    switch (fromFormat[i]) {
+      case 'YY':
+        const front = dateArr[i] < 30 ? 20 : 19;
+
+        dateKeys['YYYY'] = `${front}${dateArr[i]}`;
+        break;
+
+      case 'YYYY':
```

Comment:
it's better to move 'YYYY' and 'YY' to the constants as it will make a code cleaner